### ISSUE TITLE
Accessibility, and add start/stop play/pause translation strings for button accessibility

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/wave-on-a-string#3]
This is not required for MLL, PhET will take care of this implementation for now. (Any not-for-mll tagged issues can be ignored by the MLL team)

We should have "start" and "stop" strings for the timer, and "play" / "pause" strings for the control button that are used for screen readers. I'm keeping in the "step" string, since it should work perfectly for this purpose. We'll also want to switch string states whenever the state is toggled.
